src/norm/postgres.nim

```diff
@@ -316,3 +316,28 @@ template transaction*(dbConn; body: untyped): untyped =
 
     raise
 
+
+# One-to-Many Fetching
+proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M]) =
+  const foreignKeyFieldName: string = O.getRelatedFieldNameOn(M)
```

Comment:
I don't think it would work at compile time in such a case, which is when `getRelatedFieldNameOn` acts (now renamed to `getRelatedFieldNameTo` and likely renamed again once we can agree on a name) .